test_app/tests/lib/utils/test_encryption.py

```diff
@@ -1,6 +1,10 @@
 import pytest
+from django.test import override_settings
```

Comment:
This works, you can also just use the `settings` fixture and do `settings.FOO = 'bar'` in the test, it'll get reverted after the test finishes.

Reply:
Is there an advantage of one over the other? I've always just used `override_settings`.

Reply:
Not really, just an alternative. I like to use it because then I don't have to remember where to import the other thing from, etc. You just add `settings` as a param and go to work.